text/0101-revamping-the-sdk-performance-api.md

```diff
@@ -0,0 +1,273 @@
+| `start_timestamp` | Timestamp when span was created.                                                                                                                                                                      | String \| Float |
+| `op`              | Operation type of the span. [Standardized by Sentry spec](https://develop.sentry.dev/sdk/performance/span-operations/)                                                                                | String          |
+| `status`          | Status of the span. Sentry maps status to HTTP status codes, while OpenTelemetry has a fixed set of status'                                                                                           | String          |
+| `tags`            | Custom tags for this span.                                                                                                                                                                            | Object          |
+| `data`            | Arbitrary additional data on a span, like `extra` on the top-level event. We maintain [conventions for span data keys and values](https://develop.sentry.dev/sdk/performance/span-data-conventions/). | Object          |
+
+As you can see, the fields on the transaction/span differ in a few ways, the most noteable of which is that transactions have `name` while spans have `description`. This means that spans and transactions are not interchangeable, and users have to know the difference between the two.
+
+In addition, user's have the burden to understand the differences between `name`/`description`/`operation`. `operation` in particular can be confusing, as it overlaps with transaction `name` and span `description`. In addition, `operation` is not a required field, which means that it is not clear what the default value should be.
+
+Transactions also have no mechanism for arbitrary additional data like spans do with `data`. Users can choose to add arbitrary data to transactions by adding it to the `contexts` field (as transactions extend the event schema), but this is not obvious and not exposed in every SDK. Since contexts are already well defined in their own way, there is no way of using [Sentry's semantic conventions for span data](https://develop.sentry.dev/sdk/performance/span-data-conventions/) for transactions.
+
+### New Span Schema
+
+To simplify how performance data is consumed and understood, we are proposing a new span schema that the SDKs send to Sentry. The new span schema aims to be a superset of the [OpenTelemetry span schema](https://github.com/open-telemetry/opentelemetry-proto/blob/4967b51b5cb29f725978362b9d413bae1dbb641c/opentelemetry/proto/trace/v1/trace.proto) and have a minimal top level API surface. This also means that spans can be easily converted to OpenTelemetry spans and vice versa.
+
+The new span schema is as follows:
```

Comment:
Can we also keep `origin` https://develop.sentry.dev/sdk/performance/trace-origin/? It's handy to know what precisely created a span.

Reply:
`origin` can also be part of the span attributes if needed, but good point.